techhive-ui/src/components/Login/Login.jsx

```diff
@@ -27,34 +29,45 @@ const styles = {
       border: 'none',
       borderRadius: '5px',
       cursor: 'pointer',
-      transition: 'all 0.3s ease',
-      ':hover': {
-        background: '#0056b3'
-      }
+      transition: 'all 0.3s ease'
     }
   };
 
 function Login() {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [error, setError] = useState(null);
 
   const navigate = useNavigate(); 
 
+  const { setUser } = useContext(UserContext);
+
   const handleSubmit = async e => {
     e.preventDefault();
     try {
-      const response = await axios.post("http://localhost:3000/auth/login", { email, password });
-      console.log(response.data);
-      navigate('/');
-    } catch (err) {
-      console.error(err.response.data); 
-    }
+        const response = await axios.post("http://localhost:3000/auth/login", { email, password });
+        localStorage.setItem('user', JSON.stringify(response.data));
+        setUser(response.data.user);
+        console.log(response.data);
```

Comment:
we don't typically commit console.log statements as they're just used for debugging